examples/go/get_all_dashboards.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/looker-open-source/sdk-codegen/go/rtl"
+	v4 "github.com/looker-open-source/sdk-codegen/go/sdk/v4"
```

Comment:
super nit: seems the alias `v4` is not needed?

Reply:
Great, thanks I am updating these

Reply:
Closing this comment here, I've updated all 4 files to remove the alias. Thanks again